pkgs/development/python-modules/python-fluent/fluent-syntax.nix

```diff
@@ -0,0 +1,39 @@
+{ buildPythonPackage, fetchFromGitHub, lib, setuptools, typing-extensions }:
+
+let
+  version = "0.19.0";
+  tag = "fluent.syntax@${version}";
+
+  src = fetchFromGitHub {
+    owner = "projectfluent";
+    repo = "python-fluent";
+    rev = tag;
+    hash = "sha256-nULngwBG/ebICRDi6HMHBdT+r/oq6tbDL7C1iMZpMsA=";
+  };
+in
+buildPythonPackage {
+  pname = "fluent-syntax";
+  inherit version;
+  format = "setuptools";
```

Comment:
Does `pyproject = true` work?

Reply:
From my understanding it won't because their pyproject.toml doesn't define any information about the build system. But I never tested it.

Reply:
It will not work since the project doesn't have `pyproject.toml` in the latest release tag.

See https://github.com/NixOS/nixpkgs/pull/255937#discussion_r1659674323

Reply:
In that case, setting `pyproject = true` is actually OK:

https://github.com/NixOS/nixpkgs/blob/d41a79d80befd2d5f57d4964f31a8e50171437df/doc/languages-frameworks/python.section.md?plain=1#L167-L169

Reply:
Interesting, I'll update my PR in some time.